src/__test__/components/data-exploration/hierarchical-tree/HierarchicalTree.test.jsx

```diff
@@ -71,141 +78,399 @@ describe('HierarchicalTree', () => {
     const dropInfo = {
       dragNode: {
         ...firstChild,
-        props: { eventKey: firstChild.key },
+        pos: '0-0-0',
```

Comment:
So these `pos` properties are only available during the drag event? It's not saved along with the object?

Reply:
We aren't saving any object, you can check the reorder action, it is only sending the data we need